nifi-docker/dockerhub/pom.xml

```diff
@@ -39,7 +39,7 @@
                                     <buildArgs>
                                         <UID>1000</UID>
                                         <GID>1000</GID>
-                                        <NIFI_VERSION>1.6.0</NIFI_VERSION>
+                                        <NIFI_VERSION>1.7.0</NIFI_VERSION>
```

Comment:
Should be 1.8.0 now.

Reply:
unfortunately it cannot, since there is no released binary for 1.8 yet